sdk/src/main/java/io/opentelemetry/sdk/trace/data/SpanData.java

```diff
@@ -245,34 +245,35 @@ public static Link create(SpanContext spanContext, Map<String, AttributeValue> a
       return new AutoValue_SpanData_Link(
           spanContext,
           Collections.unmodifiableMap(new LinkedHashMap<>(attributes)),
-          ZERO_DROPPED_ATTRIBUTE_COUNT);
+          DEFAULT_ATTRIBUTE_COUNT);
```

Comment:
This should be attributes.size() correct? Same everywhere, otherwise we have negative dropped :)

Reply:
Doh! Yes - you're right :smile: I'll fix that.